tests/group_vars/all/all

```diff
@@ -0,0 +1,26 @@
+---
+streisand_ci: yes
+
+upstream_dns_servers:
+  - 8.8.8.8
+  - 8.8.4.4
+
+server_name: streisand
+
+libreswan_compilation_dependencies_trusty:
+  - bison
+  - flex
+  - gcc
+  - libcap-ng-dev
+  - libcap-ng-utils
+  - libcurl3-nss-dev
+  - libevent-dev
+  - libnspr4-dev
+  - libnss3-dev
+  - libnss3-tools
+  - libpam0g-dev
+  - libselinux1-dev
+  - libsystemd-daemon-dev
```

Comment:
On a fresh 64bit Ubuntu 16.04 machine this fails for me:
> `"No package matching 'libsystemd-daemon-dev' is available"`

```
# apt-get install libsystemd-daemon-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libsystemd-daemon-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libsystemd-dev:i386 libsystemd-dev

E: Package 'libsystemd-daemon-dev' has no installation candidate
```

Would `libsystemd-dev` be the right replacement?